handler/handler.go

```diff
@@ -1,15 +1,24 @@
 package handler
 
 import (
+	"github.com/gin-gonic/gin"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/configuration"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service"
 )
 
-var (
-	logger *configuration.Logger
-)
+var logger = configuration.NewLogger("handler")
+
+type Handler interface {
+	Save(context *gin.Context)
+	FindById(context *gin.Context)
+}
```

Comment:
Please see golang recomendation for interfaces:

https://100go.co/#interface-pollution-5
https://100go.co/#interface-on-the-producer-side-6

`Don’t design with interfaces, discover them.`

Reply:
I'll remove the interfaces and maintain it clean

Reply:
I'll remove all interfaces in the code, it will be better

Reply:
I saw your new PR with unit testing. Once you start needing to mock repositories/services or handlers you will need some kind of interface. 

`This means that it’s not up to the producer to force a given abstraction for all the clients. Instead, it’s up to the client to decide whether it needs some form of abstraction and then determine the best abstraction level for its needs.`

You will probably need some interfaces for that.

Reply:
for this PR we can keep without interfaces, if the unit test require one for mocking, I can create an interface